MSAL/src/native_auth/controllers/credentials/MSALNativeAuthCredentialsController.swift

```diff
@@ -169,11 +174,13 @@ final class MSALNativeAuthCredentialsController: MSALNativeAuthTokenController,
         }
     }
 
+    // swiftlint:disable:next function_body_length
```

Comment:
instead of adding an exception for swiftlint, can you move the claims extraction/account creation in another private method?